gnocchi/cli/metricd.py

```diff
@@ -126,6 +127,12 @@ def __init__(self, worker_id, conf):
         # This stores the last time the processor did a scan on all the sack it
         # is responsible for
         self._last_full_sack_scan = utils.StopWatch().start()
+        # Only update the list of sacks to process every
+        # metric_processing_delay
+        self._get_sacks_to_process = cachetools.func.tll_cache(
```

Comment:
this seems fine? although i should point out there already is a cache inside _get_sacks_to_process (self.group_state) i don't know if you want to keep it? unify it? *shrugs*

Reply:
Yeah but they are different cache, so I think both make sense. This TTL one is for run_watchers and hitting the coordination backend whereas the other one avoids recomputing the hash of every sack to see if assignation changed. So I think we're double-good. :)